components/ADempiere/TabManager/convenienceButtons/index.vue

```diff
@@ -304,6 +315,13 @@ export default defineComponent({
       })
     })
 
+    const fieldsListPanelInfo = computed(() => {
```

Comment:
this property is not used anywhere in the advanced search, the component itself manages and searches for its list of fields

Reply:
Resolve in next Commit  https://github.com/solop-develop/frontend-default-theme/pull/679/commits/3991aba45421a1d3e523e877ba659b7a38f27026